penguin/web/routes.py

```
@@ -5425,9 +5426,42 @@ async def execute_task_via_runmode(
 
 
 @router.get("/api/v1/token-usage")
-async def get_token_usage(core: PenguinCore = Depends(get_core)):
-    """Get current token usage statistics."""
-    return {"usage": core.get_token_usage()}
+async def get_token_usage(
+    session_id: Optional[str] = Query(default=None),
+    conversation_id: Optional[str] = Query(default=None),
+    agent_id: Optional[str] = Query(default=None),
+    core: PenguinCore = Depends(get_core),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)